File: C#/백준/Bronze/4589. Gnome Sequencing/Gnome Sequencing.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            Console.WriteLine("Gnomes:");
            for (int i = 0; i<a; i++){
                String[] aaa  = Console.ReadLine().Split();
                int[] aa = Array.ConvertAll(aaa,int.Parse);
                if((aa[0]<aa[1] && aa[1]<aa[2])||(aa[0]>aa[1] && aa[1]>aa[2])){
                    Console.WriteLine("Ordered");
                }
                else{
                    Console.WriteLine("Unordered");
                }
            }
        }
    }
}

File: Python/백준/Bronze/4589. Gnome Sequencing/Gnome Sequencing.py
a=int(input())
print("Gnomes:")
for i in range(a):
    b,c,d =map(int,input().split())
    if((b<c and c<d) or (b>c and c>d)):
        print('Ordered')
    else:
        print('Unordered')